src/components/axes/categoryAxis.ts

```diff
@@ -135,13 +129,15 @@ export module Axes {
      * @param {string[]} ticks The strings that will be printed on the ticks.
      */
     private _measureTicks(axisWidth: number, axisHeight: number, scale: Scales.Category, ticks: string[]) {
+      var axisSpace = this._isHorizontal() ? axisWidth : axisHeight;
+      var expectedRangeBand = axisSpace / (2 * scale.outerPadding() + (ticks.length - 1) * scale.innerPadding() + ticks.length);
```

Comment:
When I read inner padding, I was assuming a size (like innerPaddingWidth) (note this is exactly CSS meaning of padding). So I believe that is why the entire thing is confusing. I suggest either comment something similar to the above on the method or extract bits of the equation into variables